src/live-spine-client.ts

```diff
@@ -170,6 +159,49 @@ export class LiveSpineClient implements SpineClient {
     )
   }
 
+  async clinicalView(
+    inboundHeaders: APIGatewayProxyEventHeaders,
+    params: ClinicalViewParams
+  ): Promise<AxiosResponse> {
+    try {
+      const address = this.getSpineEndpoint(CLINICAL_VIEW_REQUEST_PATH)
+
+      const outboundHeaders = {
```

Comment:
They're not all guaranteed, but since they're top level entries in an object if they're missing we'll just end up with an `undefined` anyway. inboundHeaders is guaranteed, so I'm not sure where I can add optional chaining 🤔

Reply:
Fair enough!